src/data/uma-title.csv

```diff
@@ -59,25 +59,25 @@
 "伝説の歌姫","Legendary Diva"
 "スキルマスター","Skill Mastery"
 "技のデパート","Various Technique"
-"日本の総大将",""
+"日本の総大将","General of Japanese"
 "異次元の逃亡者","Dimensional Runaway"
 "怪鳥","Eerie Bird"
 "スーパーカー","Supercar"
 "変幻自在","Shape Shifter"
 "驀進王","Bakushin・O"
 "高速ステイヤー","High Speed Stayer"
 "不死鳥","Phoenix"
-"常識破りの女帝",""
+"常識破りの女帝","The Empress of Common Sense"
 "ミスパーフェクト","Miss Perfect"
 "帝王","Teio・Emperor"
 "破天荒","Unheard"
 "皇帝","Kotei・Emperor"
 "笑門来福",""
-"愛しき名脇役",""
+"愛しき名脇役","Lovely Supporting Role"
 "麗しき実力者",""
-"最強マイラー",""
+"最強マイラー","Strongest Miler"
 "アイドルウマ娘","Idol Girl"
-"世紀末覇王",""
+"世紀末覇王","End of Century Overlord"
```

Comment:
Supreme Ruler of the Century's End

(Or something more concise)

Reply:
well, based on the japanese people used to call TM Opera O with "End of Century Overlord/King of the end century"

Reply:
so i think that might be suitable for her

Reply:
I'm not knowledgeable on horse racing history, so i'd trust you there then.